next/components/forms/segments/NavBar/NavBar.tsx

```diff
@@ -191,85 +151,56 @@ export const NavBar = ({ className, sectionsList, menuItems, hiddenHeaderNav }:
                     onPress={register}
                     variant="negative"
                     text={t('account:menu_register_link')}
-                      size="sm"
-                      data-cy="register-button"
+                    size="sm"
+                    data-cy="register-button"
                   />
                 </div>
               )}
             </nav>
           </div>
           {/* Header bottom navigation */}
           {sectionsList && !hiddenHeaderNav && (
-            <div
-              className="m-auto hidden h-[57px] w-full max-w-screen-lg items-center justify-between border-t border-gray-200 lg:flex"
-              ref={navigationRef}
-            >
-              <ul className="flex h-full w-full items-center">
-                {sectionsList.map((sectionItem) => (
-                  <li
-                    className="h-full w-full"
-                    key={sectionItem.id}
-                    onFocus={() => setIsNavigationFocused(true)}
-                    onBlur={() => setIsNavigationFocused(false)}
-                  >
-                    <NextLink href={sectionItem.url}>
-                      <div
-                        className={cx(
-                          'text-p2-semibold flex h-full w-full cursor-pointer items-center justify-center border-b-2 transition-all hover:border-main-700 hover:text-main-700',
-                          {
-                            'border-main-700 text-main-700': isActive(sectionItem),
-                            'border-transparent': !isActive(sectionItem),
-                          },
-                        )}
-                      >
-                        {sectionItem.icon}
-                        <span className="ml-3">{t(sectionItem?.title)}</span>
-                      </div>
-                    </NextLink>
-                  </li>
-                ))}
-              </ul>
+            <div className="m-auto hidden h-[57px] w-full max-w-screen-lg items-center justify-between border-t border-gray-200 lg:flex">
+              <NavigationMenu.Root
+                value={menuValue}
+                onValueChange={setMenuValue}
+                aria-label={t('NavMenu.aria.navMenuLabel')}
+                // because of this https://github.com/radix-ui/primitives/discussions/1874 we can't directly access subelement (<div style="position: relative;")
+                // of "nav" element that NavigationMenu.List creates when used. Solution is to add custom className to NavigationMenu.Root and then use it to access subelement in index.css
+                className="nav h-full w-full"
```

Comment:
Add `grid` class and it works without `.nav` + custom css.

Reply:
:heart_eyes: